gcc/config/arc64/arc64-c.def

```diff
@@ -17,7 +17,10 @@
    <http://www.gnu.org/licenses/>.
 */
 
-//ARC_C_DEF ("__ARC_UNALIGNED__", unaligned_access)
+ARC64_C_DEF ("__ARC64_ATOMIC_0__",	(arc_atomic_option == 0))
+ARC64_C_DEF ("__ARC64_ATOMIC_1__",	(arc_atomic_option == 1))
```

Comment:
I would use a macro with value here (i.e. `__ARC64_ATOMIC__`). You can make this using:
`builtin_define_with_int_value ("__ARC64_ATOMIC__", arc64_atomic_option);` 
in `arc64-c.c` file.